lib/matplotlib/__init__.py

```diff
@@ -1545,6 +1545,41 @@ def _replacer(data, key):
 """
 
 
+def _add_data_doc(docstring, replace_names, replace_all_args):
```

Comment:
Shouldn't these be keyword arguments if they're optional? (ie. `docstring, replace_names=None, replace_all_args=False`)

Reply:
No. This is just a private helper function which does not need the flexibility of keyword args. In particular, I do not want to duplicate the defaults of `_preprocess_data`.

The docstring was just a copy from `_preprocess_data`. I've now removed the *optional*, and *default* parts.